tests/jobs/pyscal/test_pyscal.py

```diff
@@ -269,5 +280,9 @@ def test_fm_pyscal_errors(raises, dframe, runargs, tmpdir):
         with open("parameters.txt", "w") as p_file:
             p_file.write("INTERPOLATE_WO 0.1\nINTERPOLATE_GO 0.5")
 
-    with pytest.raises(raises):
+    with pytest.raises(raises) as excinfo:
         run(*runargs)
+
+    # Error message is sometimes in the exception itself, sometimes in the log.
```

Comment:
This is where I personally would split this into two tests, one where the message is in the log, and the other in the exception. But will leave it up to you.

Reply:
I think this separation of where to find error messages is more a sign of inconsistency in the code, and the fix would be to tighten up error handling in the code.